spec/services/finance/npq/statement_calculator_spec.rb

```diff
@@ -191,4 +191,32 @@
       end
     end
   end
+
+  context "when there exists contracts over multiple cohorts", with_feature_flags: { multiple_cohorts: "active" } do
+    let!(:cohort_2022) { Cohort.next || create(:cohort, :next) }
+    let!(:contract_2022) { create(:npq_contract, npq_lead_provider:, cohort: cohort_2022) }
+    let!(:statement_2022) { create(:npq_statement, cpd_lead_provider:, cohort: cohort_2022) }
+
+    before do
+      declarations = create_list(
```

Comment:
do we need `create_list` if its only one we need?

Reply:
yep, this now uses `create`